scripts/algebra-gas/README.md

```diff
@@ -0,0 +1,49 @@
+Scripts that help generate/evaluate gas parameters for generic algebra move module.
+
+## Quickstart guide
+Ensure you are on a machine with the [required spec](https://aptos.dev/nodes/validator-node/operator/node-requirements/).
+
+Ensure you have python3 and the following dependencies.
+```
+pip3 install numpy matplotlib
+```
+
+Ensure you `cd` to the repo root.
+
+Run the necessary benches.
+```
+cargo bench -p aptos-crypto -- hash/SHA2-256
+cargo bench -p aptos-crypto -- ark_bls12_381
+```
+
+Now you need an estimate of unscaled internal gas per nanosecond (referred to as `gas_per_ns` below).
+Below are some background.
+- We first model the cost of SHA2-256 evaluation as a linear function of the input size `n`: `f(n)=kn+b`,
+  then define 50 units of unscaled internal gas to be `k`.
```

Comment:
I don't understand this arbitrary choice of 50 here.

Reply:
rephrased and mentioned 50 is pre-defined.